resources/tests/conftest.py

```diff
@@ -53,3 +61,21 @@ def winter_storage_section():
 def winter_storage_place():
     winter_storage_place = WinterStoragePlaceFactory()
     return winter_storage_place
+
+
+@pytest.fixture
+def user():
+    user = UserFactory()
+    return user
+
+
+@pytest.fixture
+def staff_user():
+    user = UserFactory.build(is_staff=True)
```

Comment:
you don't have to use the `build()` method for testing, `UserFactory(is_staff=True)` is enough